components/merkledb/src/values.rs

```diff
@@ -262,6 +399,109 @@ impl BinaryValue for [u8; HASH_SIZE] {
     }
 }
 
+// Implementations for Tuples up to 12 except Tuple1 because it deserializes as is
+tuple_impls!{
```

Comment:
IMHO it would look better as following:

```rust
impl_binary_value_for_tuple!{1; T1};
impl_binary_value_for_tuple!{2; T1, T2};
impl_binary_value_for_tuple!{3; T1, T2, T3};
impl_binary_value_for_tuple!{4; T1, T2, T3, T4};
...

```

Reply:
Thank you!

I think I will do something like.